spices/SPICE-0009-external-readers.adoc

```diff
@@ -118,28 +119,18 @@ Four new message types will be added:
 ----
 /// Code: 0x100
 /// Type: Server Request
-class DiscoverModuleReaderRequest {
-  /// A number identifying this request
+class InitializeModuleReaderRequest {
+  /// A number identifying this request.
   requestId: Int
 
-  /// The scheme of the resource to discover the spec for
+  /// The scheme of the resource to initialize.
   scheme: String
```

Comment:
The reader should already know what schemes it should read when it receives a `Read*Request`. Do we need `scheme` here?

Reply:
The initialize messages only request the spec for a single scheme at a time. It would be possible to do an "all at once" sort of thing like the original proposal, but this method is actually super convenient from an implementation standpoint as it enables using the battle-tested `ConcurrentHashMap` response memoization already widely employed by the message passing API. My initial pass at the implementation that did the "all at once" discovery needed some messy locking; it would be great to pass the buck to the standard library instead.